src/VisualStudio/Core/Test.Next/Remote/RemoteHostClientServiceFactoryTests.cs

```diff
@@ -52,8 +51,11 @@ public async Task UpdaterService()
             // make sure client is ready
             using var client = await service.TryGetRemoteHostClientAsync(CancellationToken.None);
 
-            // add solution
+            // add solution, change document
             workspace.AddSolution(SolutionInfo.Create(SolutionId.CreateNewId(), VersionStamp.Default));
+            var project = workspace.AddProject("proj", LanguageNames.CSharp);
+            var document = workspace.AddDocument(project.Id, "doc.cs", SourceText.From("code"));
+            workspace.ApplyTextChanges(document.Id, new[] { new TextChange(new TextSpan(0, 1), "abc") }, CancellationToken.None);
```

Comment:
Should we add a date-like string text change to the test as well to verify that we don't hit the exception?

Reply:
Tired but doesn't repo in simple cases :( Don't know what exactly triggers the conversion.

Reply:
The original bug had `2020-01-15T00:00:00.000Z` in it -- did that not work?

Reply:
It did not.